iron-form.html

```diff
@@ -451,7 +451,9 @@
           if (this._usesCheckedInsteadOfValue(el)) {
             el.checked = this._customElementsInitialValues[i];
           } else {
-            el.value = this._customElementsInitialValues[i];
+            // The native input/textarea displays literal "undefined" when its
+            // its value is set to undefined, so default to null instead.
+            el.value = this._customElementsInitialValues[i] || null;
```

Comment:
@notwaldorf That was my bad...oops. Fixed in #148 

@rictic Please let me know if the unit tests in #148 covers the use case of your downstream client. Thanks